modules/installation-ibm-z-supported-operators.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-ibm-z-supported-operators_{context}"]
+= Supported Operators on IBM Z
+
+As a cluster administrator, you can install an Operator from the OperatorHub using the OpenShift Container Platform web console or the CLI. You can then subscribe the Operator to one or more namespaces to make it available for developers on your cluster. Following table lists all Operators that are known to be supported for {product-title} on IBM Z.
```

Comment:
s/OpenShift Container Platform/{product-title}

s/Following table/The following table

Unless you have a specific reason to include it, s/that are known to be/that are